### ISSUE TITLE
Zotxt version endpoint not found (version 5.1.2)

### ISSUE THREAD

**Issue by damiencourousse** [egh/zotxt#25]
hello, 

The last version 5.1.2 does not work, at least on my setup. 

emacs-zotxt reports the following error message:
```
\"http://127.0.0.1:23119/zotxt/items?key=1_JU4BZUWT&format=paths\" :response #0 :encoding utf-8) #<killed buffer> \"HTTP/1.0 404 Not Found
X-Zotero-Version: 5.0.95
X-Zotero-Connector-API-Version: 2
Content-Type: text/plain
\" nil curl nil)")
[error] request--callback: peculiar error: 404
[error] request-default-error-callback: http://127.0.0.1:23119/zotxt/version error
deferred error : (error "Zotxt version endpoint not found; is Zotero running and zotxt installed?")
user-error: Minibuffer window is not active
```

zotxt works fine with version 5.1.0.   I have not tested the intermediate release.

**Comment by Sepulta** [comment]
I am having the same issue as @damiencourousse, same error message for this version while 5.1.0 works.

**Comment by egh** [comment]
Thank you! Sorry for the issue - I thought I had tested this but the problem slipped through. Fixed in the 5.1.3 release.

**Comment by damiencourousse** [comment]
The 5.1.3 release has solved this.  Thanks for your quick feedback!!
